### DISCUSSION TITLE
Move note without moving focus

### DISCUSSION THREAD

**Discussion by JaCoB1123** [TriliumNext/Trilium#3392] [Ideas]
Hi,

is there a way to move a note (either via Move to... or via Drag&Drop) that keeps the sidebar-location as is?
When sorting notes from my inbox I keep having to scroll back to the inbox after each move.

Thanks :-)

**Comment by zadam** [comment]
You could open the inbox in a separate tab to avoid manual scrolling.

Another way would be assigning a keyboard shortcut to an inbox note (using e.g. `#keyboardShortcut="ctrl+h"` to come back to the inbox quickly.

**Reply by JaCoB1123** [reply]
Thanks, hoisting my Inbox in another tab helps. Just using another Tab without hoisting didn't change anything though - when switching back to the tab, the focus follows the file anyways. Could I maybe completely disable the auto-focusing of the sidebar?

Unfortunately that's a bit cumbersome since I constantly have to alternate between tabs and can't use the "Move to..." dialog, since it won't show anything outside my INBOX when hoisted.

I'll also try the shortcut idea.
